Well, the integration test seems to fail repeatedly, but that does look like a gunicorn bug:
```
Error: 4-22 12:45:18 +0000] [20101] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 116, in init_process
    self.log.close_on_exec()
  File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/gunicorn/glogging.py", line 381, in close_on_exec
    for log in loggers():
  File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/gunicorn/glogging.py", line 94, in loggers
    return [logging.getLogger(name) for name in existing]
  File "/home/runner/work/galaxy/galaxy/galaxy root/.venv/lib/python3.7/site-packages/gunicorn/glogging.py", line 94, in <listcomp>
    return [logging.getLogger(name) for name in existing]
RuntimeError: dictionary changed size during iteration
```

--- a/lib/galaxy/web_stack/gunicorn_config.py
+++ b/lib/galaxy/web_stack/gunicorn_config.py
@@ -1,11 +1,8 @@
 """
 Gunicorn config file based on https://gist.github.com/hynek/ba655c8756924a5febc5285c712a7946
 """
-import logging
 import os
 import sys
-
-log = logging.getLogger(__name__)
 
 
 def on_starting(server):

